### DISCUSSION TITLE
Is the future bright for nexus?

### DISCUSSION THREAD

**Discussion by otrebu** [graphql-nexus/nexus#894] [General]
Hello everybody! 

I have been using nexus since it is conception, I was an early adopter and used it in production for a long time ( the old version with prisma 1 ).

Working on a new exiting project I reached for one of my favourite tools: nexus.
To then find that nexus-prisma pluging is in the work now and not currently available.

I thought ok then, let's go the manual way following the guide to use nexus with prisma client without the prisma plugin.

So I did. I am having now an issue with my types not really being picked up in the resolve functions. I opened an issue here on github. Hopefully it is just gremlins in my setup.

I was looking at other issues, and there has been really little communications recently. 

So is my question, is the future of nexus bright? Are you going to continue to maintain this great project?
Or are you going to fade it out? I can see TypeGraphQL has a prisma integration...

Thanks for your work, it would be great to hear from you @Weakky  @tgriesser  @jasonkuhrt .

**Comment by tgriesser** [comment]
I'm still using it heavily / iterating on it when I can. Can't speak to the prisma integration since I'm using it with knex/redis, but I don't plan on abandoning the project any time soon.

**Reply by otrebu** [reply]
Hi @tgriesser thanks for you reply, it is really appreciated. I really truly enjoying it using nexus, and I am trying hard to stick with it.

It would be great to see a bit more communication going on the issue side of things, but I know it is time consuming. I did try to reply to some too.
Community slowly form, and you can't force them.
